app/cafe.py

```diff
@@ -0,0 +1,27 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+
+        elif visitor["vaccine"]["expiration_date"] < date.today():
```

Comment:
Use method `get` to avoid `KeyError`.
```suggestion
        elif visitor["vaccine"].get("expiration_date") < date.today():
```